### ISSUE TITLE
[Bug] [Accessibility] [Screen reader] Unnecessary messages are being announced by SR

### ISSUE THREAD

**Issue by jonaskuntoro** [ag-grid/ag-grid#12128]
### Link to reproducible scenario

https://www.ag-grid.com/example-hr/

### Describe the bug

Steps to reproduce 
1. Navigate using Screen reader
2. When you load the page
3. You can hear that there is an announcement "No rows to show" even when there are literally rows
4. The hidden message could also be reached by virtual cursor
(take a look at the attached video)

https://github.com/user-attachments/assets/7881e2d6-10db-4cd6-b6e1-7ff52a6851a0

Actual behaviour
The unnecessary messages are being announced

Expected behaviour
The unnecessary messages should be hidden with aria-hidden so it could still be announced when the actual use case is being triggered

More information
When user is navigating with Virtual cursor, there is a hidden message on top of the grid that is being announced by the SR. The message is announced because SR thinks that this message is part of the screen. This message is hidden on top of the grid. E.g. "No rows to show" when the page is loaded. The message could change based on the action done by the user, e.g. user has already navigated to the header of the column which announces "Press enter to sort the column or navigate to unselected row then there will be an announcement like "Press space to select the row", these 2 announcements/messages then will replace that "unnecessary message" on top of the grid. So if you navigate back to the top of the grid WITH VIRTUAL CURSOR, you can here those messages being announced.

### Version

Latest

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by sabahang** [comment]
Hello!

The screen recording you have sent doesn't match the URL provided.

We're keen to look into this but we need to be able to reproduce the issue you've described.

Please provide the following (as per our bug template in Github):
1. A live example (plunker, stackblitz, etc) or link to a repo showing the issue
2. A list of steps to follow to reproduce the issue
3. Descriptions of current behavior and expected behavior

We're looking forward to your response to we can investigate this further.

Thank you!

**Comment by jonaskuntoro** [comment]
hi! after a little research, it turns out that the bug is on our end. It's related to how we overlay the information when there is no row to show. Thanks for looking into it!
